### ISSUE TITLE
Accessibility issue with list item being truncated

### ISSUE THREAD

**Issue by tracytang11** [SAP/fundamental-ngx#9913]
#### Is this a bug, enhancement, or feature request?

Accessibility Issue.

##### Observed Behavior:
On accessing application using text spacing, it is observed that the last list item in the list box 'Buyers' is getting truncated.

##### Expected Behavior: 
The line height (line spacing) should be able to be adapted to at least 1.5 times the font size without loss of content or functionality. Here the list item should have been displayed correctly without getting truncated.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

angular - 14.2.4
fundamental ngx - 0.36.3

#### If this is a bug, please provide steps for reproducing it.

The issue: 
First, when the line height (line spacing) is 1 time, the list item have been displayed correctly.
<img width="1372" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/134352250/561a0936-5888-4bc6-a1c2-9442e6314075">

Second, when the line height (line spacing) is 1.5 times, the list item have been displayed uncorrectly with getting truncated.
![image](https://github.com/SAP/fundamental-ngx/assets/134352250/98bde81e-838f-41f4-ae45-09146b3e7987)

##### How to reproduce: 
1. You can reproduce this problem when you use a multi-input.
2. Prepare one of the elements whose length is close to 600. 
3. After that,change the style to { line-height: 1.5}, you can get this issue.

#### Is there anything else we should know?
About the Fundamental library component dropdown control list, because the max length of this control is 600.
We can not adapt to this length or change the max length.
So we need your help to improve it.

**Comment by droshev** [comment]
@tracytang11 Any update?

**Comment by droshev** [comment]
no activities => closing the issue
